src/main/java/seedu/duke/commands/EditCheatSheetCommand.java

```diff
@@ -0,0 +1,33 @@
+package seedu.duke.commands;
+
+import seedu.duke.editor.TextEditor;
+import seedu.duke.ui.UI;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static seedu.duke.common.Messages.MESSAGE_FILE_DOES_NOT_EXIST;
+
+public class EditCheatSheetCommand extends DeleteCheatSheetCommand {
```

Comment:
Is there a reason why this command needs to extend the delete command?

Reply:
Edit uses most methods from delete. Edit contains an overridden method and an additional one to perform its function.